tests/network/bindingplugin.go

```diff
@@ -100,6 +104,65 @@ var _ = SIGDescribe("[Serial]network binding plugin", Serial, decorators.NetCust
 			Expect(vmi.Status.Interfaces[0].MAC).To(Equal(macAddress))
 		})
 	})
+
+	Context("macvtap", func() {
+		const (
+			macvtapNetworkConfNAD = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s", "annotations": {"k8s.v1.cni.cncf.io/resourceName": "macvtap.network.kubevirt.io/%s"}},"spec":{"config":"{ \"cniVersion\": \"0.3.1\", \"name\": \"%s\", \"type\": \"macvtap\"}"}}`
+			macvtapBindingName    = "macvtap"
+			macvtapLowerDevice    = "eth0"
+			macvtapNetworkName    = "net1"
+		)
+
+		createMacvtapNetworkAttachmentDefinition := func(namespace, networkName, macvtapLowerDevice string) error {
+			macvtapNad := fmt.Sprintf(macvtapNetworkConfNAD, networkName, namespace, macvtapLowerDevice, networkName)
+			return createNetworkAttachmentDefinition(kubevirt.Client(), networkName, namespace, macvtapNad)
+		}
+
+		BeforeEach(func() {
+			Expect(createMacvtapNetworkAttachmentDefinition(testsuite.GetTestNamespace(nil), macvtapNetworkName, macvtapLowerDevice)).
+				To(Succeed(), "A macvtap network named %s should be provisioned", macvtapNetworkName)
+		})
+
+		BeforeEach(func() {
+			err := libkvconfig.WithNetBindingPlugin(macvtapBindingName, v1.InterfaceBindingPlugin{DomainAttachmentType: v1.Tap})
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		Context("can run a virtual machine with one macvtap interface", func() {
+			var vmi *v1.VirtualMachineInstance
+			var chosenMAC string
+			var nodeList *k8sv1.NodeList
+
+			BeforeEach(func() {
+				nodeList = libnode.GetAllSchedulableNodes(kubevirt.Client())
+				Expect(nodeList.Items).NotTo(BeEmpty(), "schedulable kubernetes nodes must be present")
+				nodeName := nodeList.Items[0].Name
```

Comment:
I do not understand why this is needed.
We do not check this for other tests, what is special about this one?

If there are no scheduled nodes, it should fail on timing out to create the VM.
I do not need to know about these details unless you tell me it is a must.

Reply:
It is nit needed, removed.